src/framework/audio/internal/worker/mixer.cpp

```diff
@@ -20,19 +20,27 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 #include "mixer.h"
+
+#include "async/async.h"
 #include "log.h"
+
+#include <limits>
+
 #include "internal/audiosanitizer.h"
+#include "internal/audiothread.h"
+#include "internal/audiomathutils.h"
+#include "audioerrors.h"
 
+using namespace mu;
 using namespace mu::audio;
+using namespace mu::async;
 
 Mixer::Mixer()
 {
-    ONLY_AUDIO_WORKER_THREAD;
 }
 
 Mixer::~Mixer()
 {
-    ONLY_AUDIO_WORKER_THREAD;
```

Comment:
I recommend keeping this and making it so that the mixer is created in the thread in which it should be used.